This patch is part of a series of patches for festival for
Debian GNU/Linux, to correct spelling

Index: b/doc/festival_client.head
===================================================================
--- a/doc/festival_client.head
+++ b/doc/festival_client.head
@@ -15,7 +15,7 @@
 option 
 .BI --server
 
-festival-clinet allows waveforms ot be generated faster as not
+festival-client allows waveforms to be generated faster as not
 time is required for start up.  See the festival manual for
 more details on recommended use.
 
